ci/prow/config.yaml

```diff
@@ -531,7 +531,7 @@ presubmits:
         args:
         - "./test/presubmit-tests.sh"
         - "--run-test"
-        - "./test/e2e-upgrade-tests.sh --istio-version 1.2-latest --no-mesh"
+        - "./test/e2e-upgrade-tests.sh --istio-version 1.4-latest --no-mesh"
```

Comment:
I was not sure whether we should use Istio 1.3 or 1.4 for pull-knative-serving-integration-tests.
I think latest (Istio 1.4) is better, but I will change it to 1.3 if we should preserve 1.3.

Reply:
Never mind. 1.3 should be used, I think.